File: code/src/main/java/com/withoutcat/petwish/controller/HealthzController.java
package com.withoutcat.petwish.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HealthzController {

  @GetMapping("/healthz")
  public String healthzCheck() {
    return "这里是withoutcat的健康检查";
  }
}

File: code/src/main/kotlin/com/withoutcat/petwish/controller/HealthzController.kt
package com.withoutcat.petwish.controller

import org.springframework.http.HttpStatus
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthzController {

    @GetMapping("/")
    fun defaultDomainResponse(): ResponseEntity<Void> {
        return ResponseEntity.status(HttpStatus.FOUND)
            .header("Location", "/welcome.html")
            .build()
    }
}